### PR TITLE
INT-3877: add new pog payway

### PR DESCRIPTION
**Pull request by SergeKoTnk**

### CODE REVIEW

File: data/payment_providers.json [hunk 1]
```diff
@@ -18437,6 +18437,19 @@
       "payment_card"
     ]
   },
+  {
+    "status": "active",
```
Comment (TonyTramb) [inline data/payment_providers.json:18441]: https://www.payway.com/

Comment (SergeKoTnk) [reply data/payment_providers.json:18441]: fixed

Comment (TonyTramb) [reply data/payment_providers.json:18441]: Logo + icon ещё

Comment (SergeKoTnk) [reply data/payment_providers.json:18441]: added

**Review by TonyTramb — APPROVED** [review]

**Pull request merged**